cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -348,21 +348,30 @@ public static Resources getResources(Map<String, String> args, @Nullable Resourc
   }
 
   /**
-   * Get the profile id for the given program, given arguments for a run. All non-workflow program types will use the
-   * native profile. Workflow program types will use whatever profile is specified in its arguments, or the native
-   * profile if none is specified.
+   * Determine if compute profile is allowed to be applied on the given program type.
```

Comment:
Seems like a wrong modification of the javadoc of this method??

Reply:
Thanks :)